packages/cli/src/utils/checkMissingAppInfo.js

```diff
@@ -0,0 +1,7 @@
+module.exports = async function (app: Object) {
+  const requiredFields = ['title', 'description', 'app_category', 'intention', 'role'];
+  const missingRequiredFields = requiredFields.filter(field => app[field] == null);
+  throw new Error(
+    `Your integration is missing required info (${missingRequiredFields.join(', ')}). Please, run "zapier register" to add it.`
+  );
```

Comment:
This function always throws an `Error` (even if the app has all required fields). Can you please update this to check if any required fields are missing?

Here's a suggestion: 
```suggestion
  if (missingRequiredFields.length) {
    throw new Error(
      `Your integration is missing required info (${missingRequiredFields.join(', ')}). Please, run "zapier register" to add it.`
    );
  }
```

Reply:
Of course! 😄